src/app/blog/index.html

```diff
@@ -4,6 +4,18 @@
 	windowTitle="SKY UX Blog"
 	showTableOfContents="true">
 	
+	<article class="blog-post">
+		<header>
+			<stache-page-anchor>
+				Design update — June 2020
+			</stache-page-anchor>
+		</header>
+		<section>
+			<h3 class="sky-subsection-heading">Recently completed</h3>
```

Comment:
Given that the description says `in the process of`, should this header be `In the works`? Or should we update the description to use past tense? (i.e. `We addressed some accessibility issues.` as per Denise's suggested wording update.)

Reply:
updated both the heading and the text